tsconfig.base.json

```diff
@@ -14,6 +14,9 @@
 		// Allow JS in order to check the JSTokenizer
 		"allowJs": true,
 
+		// Allow parsing of renamed html (to jsx) files
+		"jsx": "preserve",
```

Comment:
In order not to transform the inpout of the JSX file, we need to set this option to "preserve".
More information can be seen here: https://www.typescriptlang.org/docs/handbook/jsx.html#basic-usage

Reply:
This config is not used in the type checker, but for the UI5 linter sources.

You need to configure this in `src/detectors/typeChecker/index.ts`

Reply:
Thanks!